### ISSUE TITLE
Need support for screen readers

### ISSUE THREAD

**Issue by jimeng** [sciactive/pnotify#293]
We use PNotify to make users aware of success or failure of their actions and for other purposes.  The information is generally essential for people to understand what has occurred and what they need to do next.  But PNotify pop-ups are not seen by screen readers.  Relying on PNotify then causes our webapp to fail for people using a screen reader.  We will find other ways to deal with this in the short term, but for the future, please build accessibility into PNotify.

**Comment by jimeng** [comment]
Duplicate of #184

**Comment by hperrin** [comment]
The alerts do have aria tags that should cause them to be read by screen readers. If it's not working, can you open a new issue. Thanks.
